src/tool/subcommands/api_cmd.rs

```diff
@@ -177,41 +183,51 @@ pub enum RunIgnored {
     All,
 }
 
+/// Brief description of a single method call against a single host
 #[derive(Debug, Clone, Copy, PartialOrd, Ord, PartialEq, Eq, Hash)]
-enum EndpointStatus {
-    // RPC method is missing
+enum TestSummary {
```

Comment:
nonblocking nit: I'm unsure if I like `TestSummary` naming, but I can't figure out a better one. This kinda mixes several different concepts together, like actual response errors, check errors and an actual valid status. Perhaps something like `TestStatus` would be more intelligible. In my head summary is something that could potentially have several properties, but here we always have only one outcome.